examples/app/random.rs

```diff
@@ -0,0 +1,96 @@
+use bevy::prelude::*;
+use rand::{prelude::IteratorRandom, rngs::SmallRng, Rng, SeedableRng};
+
+/// This example illustrates how to use entropy to control randomness in bevy.
+/// We randomly choose a coin to toss and record the result.
+///
+/// Because all random number generators are seeded from the same "world seed",
+/// this program is deterministic.
+struct Coin;
+
+struct Toss(SmallRng);
+
+#[derive(Debug, PartialEq)]
+enum Face {
+    Heads,
+    Tails,
+}
+
+fn main() {
+    // This is a bad seed but is used for illustrative purposes.
```

Comment:
It's actually a very *good* seed for many uses (like scientific computing), because it shows that it wasn't cherrypicked.

Reply:
It may be a bad seed in that it is possible that it could causes the prng to produce bad output. I believe many prng algorithms need roughly half of all bits to be 1 to produce good results or something like that. This seed has just 1/4 of that. Something like the first digits of pi or e would make it just as unlikely that it was cherrypicked while being a much better seed.

Reply:
TIL. If that applies here, that should be documented somewhere.

Reply:
Will change this and fill out comments.